web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -273,6 +275,20 @@
             ngeoDecorateLayer(olLayer);
             olLayer.displayInLayerManager = true;
 
+            var unregisterEventKey = olLayer.getSource().on('tileloaderror',
```

Comment:
I would also unregister this event on a tileloadsuccess, how your code behave if the layer is served but one tile fails ?
I have some layers where tile loading fails for some extents, or some resolutions, but mostly WMTS, and your work works only for WMS so maybe we don't need to handle that case for now.